scripts/schema.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE offenders
+(
+    id int not null IDENTITY(1,1),
+    offender_id integer,
+    gender nchar(1),
+    region smallint,
+    agency nvarchar(30),
+    supervisor_id nchar(8),
+    supervisor_name nvarchar(35),
+    agent_id nchar(8),
+    agent_name nvarchar(40),
+    offender nvarchar(60),
+    date_of_birth datetime,
+    race nvarchar(30),
+    legal_status nvarchar(10),
+    legal_status_code nchar(1),
+    legal_status_description nvarchar(30),
+    supervision_start_date datetime,
+    offender_location nvarchar(40),
+    address_start_date datetime,
+    address nvarchar(50),
+    unit nvarchar(15),
+    city nvarchar(30),
+    state nchar(2),
+    zip integer(5),
```

Comment:
Should this be `zip integer`? I got "Cannot specify a column width on data type int" when I tried to run this script.